pipenv/cli/command.py

```diff
@@ -721,20 +722,23 @@ def clean(ctx, state, dry_run=False, bare=False, user=False):
     context_settings=subcommand_context_no_interspersion,
 )
 @common_options
-@argument("args", nargs=-1)
-@pass_state
-def scripts(state, args):
+def scripts():
     """Lists scripts in current environment config."""
     from ..core import project
-    if not project:
-        echo(u"project not found", err=True)
-        exit(1)
+
+    if not project.pipfile_exists:
+        echo("No Pipfile present at project home.", err=True)
+        sys.exit(1)
     scripts = project.parsed_pipfile.get('scripts', {})
-    rpt = u"command\tscript\n"
-    for k, v in scripts.items():
-        rpt += u"{0}\t{1}".format(k, v)
-    echo(rpt)
-    return 0
+    first_column_width = max(len(word) for word in ["Command"] + list(scripts.keys()))
+    second_column_width = max(len(word) for word in ["Script"] + list(scripts.values()))
+    lines = ["{}  Script".format("Command".ljust(first_column_width))]
+    lines.append("{}  {}".format("-" * first_column_width, "-" * second_column_width))
+    lines.extend(
+        "{}  {}".format(name.ljust(first_column_width), script)
+        for name, script in scripts.items()
+    )
+    echo(os.linesep.join(fix_utf8(line) for line in lines))
```

Comment:
I think we can just use `\n`. `os.linesep` is for writing raw files, `sys.stdout` handles it automatically.